src/lib/operators/aggregate_hash.cpp

```diff
@@ -584,11 +665,17 @@ void AggregateHash::_aggregate() {
             // Not grouped by anything, simply count the number of rows
             results.resize(1);
             results[0].aggregate_count += input_chunk_size;
+
+            // We need to set a RowID because the default value (NULL_ROW_ID) would later be skipped. This RowID would
+            // otherwise be used to recreate the GROUP BY values, but as we are not grouping, it will be ignored.
```

Comment:
Don't get the comment. First it's said that NULLs are skipped, than is used to recreate the GROUP BY values? I am confused.

Reply:
> We need to set any RowID because the default value (NULL_ROW_ID) would later be skipped. As we are not reconstructing the GROUP BY values later, the exact value of this row_id does not matter, as long as it not NULL_ROW_ID.

Better?

Reply:
Jep.